lib/PSR/Storage/SQLite.hs

```diff
@@ -53,10 +56,22 @@ mkStorage conn = do
     addExecutionEvent blockHeader ExecutionEventPayload{..} = do
         withTransaction conn $ do
             blockId <- getOrCreateBlockId blockHeader
-            execute
+            let
+                params =
+                    [ ":block_id" := blockId
+                    , ":transaction_hash" := transactionHash
+                    , ":script_hash" := scriptHash
+                    , ":script_name" := scriptName
+                    , ":trace_logs" := traceLogs
+                    , ":ex_units" := exUnits
+                    , ":eval_error" := evalError
+                    ]
+            executeNamed
                 conn
-                "INSERT INTO execution_event (block_id, transaction_hash, script_hash, name, trace) values (?, ?, ?, ?, ?)"
-                (blockId, transactionHash, scriptHash, scriptName, trace)
+                "INSERT INTO execution_event \
+                \ (block_id, transaction_hash, script_hash, name, trace_logs, ex_units, eval_error) \
+                \ values (:block_id, :transaction_hash, :script_hash, :script_name, :trace_logs, :ex_units, :eval_error)"
+                params
```

Comment:
I'm having a hard time reviewing raw embedded sql. I trust you w.r.t the correctness of this :-)